client/src/components/Profile.jsx

```diff
@@ -8,17 +8,29 @@ class Profile extends Component {
 
     state = {
         user: {},
-        page_id: ""
     };
 
     componentDidMount() {
-        this.setState({ page_id: this.props.match.params.profile_id });
         request('/profile/' + this.props.match.params.profile_id)
-        .then(json => {
-            this.setState({ user: json });
+        .then(data => {
+            this.setState({ user: data });
+            if (sessionStorage['User-nickname'] === this.state.user.nickname) {
+                sessionStorage.setItem("User-nickname",
+                                        this.state.user.nickname);
```

Comment:
Can you tell me what is happening here? (17-19 line).
`
If a === b {
a = b
}
`